vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.html

```diff
@@ -628,598 +668,722 @@
+}
+
+function makeModuleLinksEditable(moduleLinkName, moduleLinkId, rotation, selectedModuleId, posXEdit, posYEdit, displayLinkId, linkType) {
+    
+    selectedModuleLinkId=moduleLinkId;
+    storeX=posXEdit;
+	storeY=posYEdit;
+	$("#moduleLinkInfoLabel").text(moduleLinkName);
+	$("#moduleLinkDisplayId").val(displayLinkId);
+	$("#moduleLinkInfoLabelEdit").text(moduleLinkName);
+	$("#moduleLinkLabelEdit").val(moduleLinkName);
+	$("#moduleLinkId").val(moduleLinkId);
+	$("#moduleLinkIdValueEdit").val(moduleLinkId);
+	$("#moduleLinkRotationEdit").val(rotation);
+	$('#typeEdit option[value="'+linkType+'"]').attr("selected", "selected");
+	$('#moduleLinkIdEdit option[value="'+selectedModuleId+'"]').attr("selected", "selected");
+	resetHighlighting();     
+	$('[data-link-id="' + moduleLinkId + '"]').css("color", "#c1bb88");
+	$('div[data-link-id="' + moduleLinkId + '"]').removeClass("alert-primary");
+	$('div[data-link-id="' + moduleLinkId + '"]').addClass("alert-warning");
+	$('img[data-link-id="' + moduleLinkId + '"]').css("border", "solid 1px #c1bb88");
+	
+	$("#bgImage").on("click", function(e){
+		e.preventDefault();
+		var posX = $(this).position().left;
+		var posY = $(this).position().top;    
+		storeX = e.pageX - $(this).offset().left;
+		storeY = e.pageY - $(this).offset().top;
+		showModuleLinkEdit(editModuleLinkInfo(), false, storeX + $("#space").scrollLeft(),storeY + $("#space").scrollTop());
+	});
+	hideLinkInfoTabs();
+	$("#space").css("display", "-webkit-box");
+    $("#editModuleLinkInfo").css("display", "flex");
+}
+
+function makeExternalLinksEditable(linkName, linkId, externalLinkURL, posXEdit, posYEdit, displayLinkId, linkType) {
+    selectedExternalLinkId=linkId;
+    storeX=posXEdit;
+	storeY=posYEdit;
+    $("#externalLinkInfoLabel").text(linkName);
+    $("#externalLinkId").val(linkId);
+	$("#externalLinkDisplayId").val(displayLinkId);
+	$("#externalLinkInfoLabelEdit").text(linkName);
+	$("#externalLinkLabelEdit").val(linkName);
+	$("#externalLinkIdValueEdit").val(linkId);
+	$("#externalLinkURLEdit").val(externalLinkURL);
+	$('#extTypeEdit option[value="'+linkType+'"]').attr("selected", "selected");
+    resetHighlighting();
+          
+    $('[data-link-id="' + linkId + '"]').css("color", "#c1bb88");
+    $('div[data-link-id="' + linkId + '"]').removeClass("alert-primary");
+    $('div[data-link-id="' + linkId + '"]').addClass("alert-warning");
+    $('img[data-link-id="' + linkId + '"]').css("border", "solid 1px #c1bb88");
+    
+    $("#bgImage").on("click", function(e){
+		e.preventDefault();
+		var posX = $(this).position().left;
+		var posY = $(this).position().top;    
+		storeX = e.pageX - $(this).offset().left;
+		storeY = e.pageY - $(this).offset().top;
+		showExternalLinkEdit(editExternalLinkInfo(), false, storeX + $("#space").scrollLeft(),storeY + $("#space").scrollTop());
+	});
+    hideLinkInfoTabs();
+    $("#space").css("display", "-webkit-box");
+    $("#editExternalLinkInfo").css("display", "flex");
+}

```

Comment:
same as above, this looks like a lot of repeating code that can be made more DRY